composer.json

```diff
@@ -42,7 +42,7 @@
         "ext-zlib": "*",
         "ext-sockets": "*",
         "appwrite/php-clamav": "1.1.*",
-        "appwrite/php-runtimes": "dev-refactor",
+        "appwrite/php-runtimes": "0.7.2",
```

Comment:
shouldn't it already be `apwrite/runtimes`?

Reply:
I did try it, but for some reason my composer couldn't find it . I didn't spend much time trying to debug it.